dt-assets/parts/modals/modal-help.php

```diff
@@ -289,25 +289,25 @@
     if ( is_archive() ):?>
         <!--  Filters Tile - left side -->
         <div class="help-section" id="filters-help-text" style="display: none">
-            <h3><?php echo esc_html_x( "Default and Custom Filters", 'Optional Documentation', 'disciple_tools' ) ?></h3>
+            <h3><?php esc_html_e( "Default and Custom Filters", 'disciple_tools' ) ?></h3>
             <p><?php echo esc_html_x( "Use these filters to focus in on the contacts or groups you are responsible for. If the default filters do not fit your needs you can create your own custom filter.", 'Optional Documentation', 'disciple_tools' ) ?></p>
         </div>
 
         <!--  Contacts List Tile -->
         <div class="help-section" id="contacts-list-help-text" style="display: none">
-            <h3><?php echo esc_html_x( "Contacts List Tile", 'Optional Documentation', 'disciple_tools' ) ?></h3>
+            <h3><?php esc_html_e( "Contacts List Tile", 'disciple_tools' ) ?></h3>
```

Comment:
Let's not remove the "optional documentation" context unless it is creating duplicate translation strings.
To be applied to the other changes as well.

Reply:
> Let's not remove the "optional documentation" context unless it is creating duplicate translation strings.
> To be applied to the other changes as well.

I agree. That's why I went aded or removed or modified the contexts - so to combine the strings that were needing to be translated more than once.

Reply:
No. The context was removed so that the heading could be translated and not be optional. I understood that headings would be more useful that the descriptions. It and and other headings are not difficult to translate, and the more translated the better.

Reply:
Thank you for explaining.
Our goal was to have the documentation (especially strings that only show up on theme pages) be easily skipped with the "Optional Documentation" context. Just to reduce the number of them.